app/models/language_term.rb

```diff
@@ -15,6 +15,79 @@
 class LanguageTerm
   class LookupError < Exception; end
 
+  class Iso6391
```

Comment:
It seems like it might be possible to DRY this up by making this a subclass of `LanguageTerm` and making some constants that get overridden (e.g. controlled vocabulary URL and file path).

If you do go this route, you should be careful that the class instance variable `@@map` isn't shared between the two implementations.